gui/src/main/java/org/verapdf/gui/ValidateWorker.java

```diff
@@ -104,14 +103,15 @@ protected BatchSummary doInBackground() {
 		// TODO: change "== 1" to "> 0" when HTML report will work in case of multiply files
 		if (this.batchSummary.getValidPdfaCount() + this.batchSummary.getInvalidPdfaCount() == 1) {
 			writeHtmlReport();
+			writePDFReport();
```

Comment:
What if there will be some exception while creating HTML report? Add if statement for checking that HTML report was created please.